base/Ability.ts

```diff
@@ -100,6 +100,8 @@ export class Ability {
     public affects_pp: boolean;
     public has_animation_variation: boolean;
     public can_be_mirrored: boolean;
+    public field_color: number;
```

Comment:
Please remove these color properties. We don't need them. As we can change the wave color, we will let it have any color.

Reply:
These colors aren't for the wave effect, they are for the background when you use a field psynergy. It usually issues a random color, but some abilities like pound always issue the same color

Reply:
Let us make pound to issue different colors too. If we are keeping these properties, the right place for them is inside their classes, since each field psynergy has its own class.

Reply:
I was thinking about a solution for this. Let us use one color just as the game. Set these two properties as members of FieldAbilities, when constructing pound, pass these values to FieldAbilities construction (super). When calling `tint_map_layers`, check whether these properties are set. The color and the intensity can be constants in the pound class.